apps/docs/src/components/docs-background.tsx

```
@@ -0,0 +1,97 @@
+      backgroundImage:
+        'radial-gradient(49.63% 57.02% at 58.99% -7.2%, hsl(var(--color-fd-primary)/0.1) 39.4%, transparent 100%)',
+    }}
+  >
+    <svg
+      width='790'
+      height='640'
+      viewBox='0 0 790 718'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+      className='-top-16 -translate-x-1/2 absolute left-1/2 pl-48'
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)